### PR TITLE
Cache

### PR DESCRIPTION
**Pull request by Izzette**

### CODE REVIEW

File: UnanimusProject/src/com/sjgilbert/unanimus/parsecache/ParseCache.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    public static void init(Object ... objects) {
+        parseCache = new ParseCache<>(objects);
+    }
+
+    public static void free() {
+        parseCache = null;
+    }
+
+    public static boolean isInit() {
+        return (parseCache == null);
```
Comment (Izzette) [inline UnanimusProject/src/com/sjgilbert/unanimus/parsecache/ParseCache.java]: Correction: `(parseCache != null);`

**Comment by sjgilbert** [comment]
looks good, ready to merge it?

**Comment by Izzette** [comment]
Not quite, just gotta fix this one submodule thing

**Comment by Izzette** [comment]
Okay, it's good now.  Just make sure to run this after you check it out the first time.

``` bash
[user@localhost Unanimus]$ git submodule init
[user@localhost Unanimus]$ git submodule sync
[user@localhost Unanimus]$ git submodule update
```

**Pull request merged**